app/_src/gateway/production/monitoring/healthcheck-probes.md

```diff
@@ -127,11 +157,32 @@ readinessProbe:
     periodSeconds: 5
 ```
 
+{:.note}
+> **Note:**  Failure to set an `initialDelaySeconds` may result in {{site.base_gateway}} entering a crash loop, as it requires a short time to fully load the configuration. The time to delay can depend on the size of the configuration.
+
+## Using a readiness check in version 3.2 or lower.
+
+The `/status/ready` endpoint was added in version 3.3, meaning versions 3.2 and lower do not benefit from this built-in readiness endpoint. However a workaround exists which is strongly recommended:
+
+1. Configure a new route in {{site.base_gateway}} with the path uniqely set for this purpose. For example, a path of `/health/ready` can be used. No need for a service, only the route is required.
+2. Configure the Request Termination plugin to respond to requests on that route with a HTTP 200 status code.
+
+Note that in this workaround, the port to send health-check requests to will be the proxy port (8000 & 8443 by default) instead of the status API port.
```

Comment:
```suggestion
{:.note}
> **Note:** In this workaround, the port to send health check requests to is the proxy port (8000 & 8443 by default) instead of the status API port.
```